src/javafundamentals/collections/list/Ex04DeleteElements.java

```diff
@@ -1,9 +1,28 @@
 package javafundamentals.collections.list;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 /**
  * Create an List with numbers from 1 to 10
  * Delete the number 7 and print the list
  */
 
 public class Ex04DeleteElements {
+    public static void main(String[] args) {
+        List<Integer> myList = new ArrayList<>();
+        createList(myList);
+        myList.removeAll(Collections.singleton(7));
```

Comment:
Nice! you used the `Collections.singleton()` method to create a list of one element!
As well you could use `myList.remove(new Integer(7))` since we already know that there will be only one 7 since we created the list. But your solution cover more cases! nice!